opencti-platform/opencti-front/lang/front/en.json

```diff
@@ -2163,7 +2163,7 @@
   "Level": "Level",
   "color": "color",
   "Force now": "Force now",
-  "name": "name",
+  "name": "Name",
```

Comment:
I don't think name should be translated to Name because it changes the case. (Knowing that the 'name' and 'description' label will be ok in 4939